pom.xml

```diff
@@ -933,7 +933,33 @@
                 <module>splice_spark</module>
             </modules>
         </profile>
-
+        <profile>
+            <id>hdp2.6.4</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>hdp2.6.4</value>
+                </property>
+            </activation>
+            <properties>
+                <envClassifier>hdp2.6.4</envClassifier>
+                <envHbase>hbase1.1.2.2.6</envHbase>
+                <hadoop.version>2.7.3.2.6.4.0-91</hadoop.version>
```

Comment:
How is 2.7.3.2.6.4.0-91 chosen? Is this the latest stable version?

There are other version available from Hortonworks repo:
http://repo.hortonworks.com/content/repositories/releases/org/apache/hadoop/hadoop-main/